build.bat

```diff
@@ -1,33 +1,15 @@
 @echo off
 
-.paket\paket.bootstrapper.exe
```

Comment:
see L3- bootstrapper needs to run before paket.exe (and build.fsx refs some of the stuff that installs)

Reply:
Paket.bootstrap has been renamed to paket.exe (magic mode). So there is no need to call the bootstrap first.

Reply:
ah, ok, missed that, great